docs/book/ceph-authorization.md

```diff
@@ -0,0 +1,1197 @@
+  name: rook-ceph-system
+  namespace: rook-ceph-system
+  labels:
+    operator: rook
+    storage-backend: ceph
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: rook-ceph-system
+  namespace: rook-ceph-system
+  labels:
+    operator: rook
+    storage-backend: ceph
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: rook-ceph-system
+subjects:
+- kind: ServiceAccount
+  name: rook-ceph-system
+  namespace: rook-ceph-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: rook-ceph-global
+  namespace: rook-ceph-system
+  labels:
+    operator: rook
+    storage-backend: ceph
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rook-ceph-global
+subjects:
+- kind: ServiceAccount
+  name: rook-ceph-system
+  namespace: rook-ceph-system
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: rook-ceph-operator
+  namespace: rook-ceph-system
+  labels:
+    operator: rook
+    storage-backend: ceph
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: rook-ceph-operator
+    spec:
+      serviceAccountName: rook-ceph-system
+      containers:
+      - name: rook-ceph-operator
+        image: openpolicyagent/rook-operator:latest
+        args: ["ceph", "operator"]
+        volumeMounts:
+        - mountPath: /var/lib/rook
+          name: rook-config
+        - mountPath: /etc/ceph
+          name: default-config-dir
+        env:
+        - name: ROOK_ALLOW_MULTIPLE_FILESYSTEMS
+          value: "false"
+        - name: ROOK_LOG_LEVEL
+          value: "INFO"
+        - name: ROOK_MON_HEALTHCHECK_INTERVAL
+          value: "45s"
+        - name: ROOK_MON_OUT_TIMEOUT
+          value: "300s"
+        - name: ROOK_DISCOVER_DEVICES_INTERVAL
+          value: "60m"
+        - name: ROOK_HOSTPATH_REQUIRES_PRIVILEGED
+          value: "false"
+        - name: ROOK_ENABLE_SELINUX_RELABELING
+          value: "true"
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+      volumes:
+      - name: rook-config
+        emptyDir: {}
+      - name: default-config-dir
+        emptyDir: {}
+```
+
+Create the operator:
+
+```bash
+kubectl create -f operator.yaml
+```
+
+Verify that `rook-ceph-operator`, `rook-ceph-agent`, and `rook-discover` pods are in the `Running` state.
+
+```bash
+kubectl -n rook-ceph-system get pod
+```
+
+> The Rook operator image `openpolicyagent/rook-operator:latest` used in the tutorial supports the latest version of Ceph (`nautilus`). Rook currently does not support the latest version of Ceph. See [this](https://github.com/rook/rook/issues/2475) for more details.  
```

Comment:
[nit] I _think_ accessibility of the docs would be improved a little by naming the link properly:
```suggestion
> The Rook operator image `openpolicyagent/rook-operator:latest` used in the tutorial supports the latest version of Ceph (`nautilus`). Rook currently does not support the latest version of Ceph. See [this Github issue](https://github.com/rook/rook/issues/2475) for more details.  
```
(People using screen readers won't see the URL -- they're not hovering over the link with a mouse, I believe.)